src/logger.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
Simple logger class, log level should be added here (or not at all, we could just grep through the logs)